package.json

```diff
@@ -1,11 +1,35 @@
 {
   "name": "dctl",
-  "version": "1.0.3",
+  "version": "4.0.0",
```

Comment:
damn is this big of a jump necessary? I understand the underlying sdk is 4.0.0

Reply:
Yeah, just making it match the sdk to hopefully be a little clearer what sdk is mapped to the dctl. I'm hoping to keep both the major.minor versions of dctl and it's corresponding sdk the same so it's more clear what you're using.